docs/csharp/language-reference/toc.yml

```diff
@@ -2345,7 +2343,3 @@ items:
       href: ./compiler-messages/cs1610.md
     - name: CS1712
       href: ../misc/cs1712.md
-- name: C# conceptual documentation
```

Comment:
The reason I removed them is because all the other nodes are really part of "C# reference documentation", but these two aren't, and yet they're at the same level.

Reply:
That's why I put them at the root level (on all three C# based TOCs)

It's not critical, but it was a consistent navigation element.